content/microcontrollers/feather-nrf52840.md

```diff
@@ -57,3 +59,42 @@ Once you have updated your Adafruit Feather nRF52840 board the first time, after
 You can use the USB port to the Adafruit Feather nRF52840 as a serial port. `UART0` refers to this connection.
 
 Bluetooth support is now available for the Adafruit Feather nRF52840 board. See https://github.com/tinygo-org/bluetooth for more information.
+
+## Updating the UF2 bootloader
+
+This board uses a UF2 bootloader created by Adafruit: https://github.com/adafruit/Adafruit_nRF52_Bootloader
+
+We recommend bootloader version 0.4.0 or above. You can check what version is installed on your board by double-clicking the button on the board to launch the bootloader. When you do this, a USB volume that should automatically be mounted on your computer. Check the file named "INFO_UF2.TXT" on that drive. The bootloader firmware version should be listed in that file, for example:
+
+```
+UF2 Bootloader 0.4.0 lib/nrfx (v2.0.0) lib/tinyusb (0.6.0-272-g4e6aa0d8) lib/uf2 (remotes/origin/configupdate-9-gadbb8c7)
+Model: Adafruit CLUE nRF52840
+Board-ID: nRF52840-CLUE-revA
+SoftDevice: S140 version 6.1.1
+Date: Jan 19 2021
+```
+
+To update the bootloader, you will need to install the `adafruit-nrfutil` program. 
+
+You can install it by running:
+
+```shell
+pip3 install --user adafruit-nrfutil
+```
+
+Once you have installed the `adafruit-nrfutil` program, download the firmware here: 
+https://github.com/adafruit/Adafruit_nRF52_Bootloader/releases/download/0.4.0/feather_nrf52832_bootloader-0.4.0.zip
+
+Unzip the files in this zip file and save them to a convenient location. 
+
+Plug in your board to your computer's USB port.
+
+Now we are ready to update the firmware. Run a command something like the following, adjusting for any difference based on where you have saved the files, and what your serial port is named:
+
+```shell
+ adafruit-nrfutil --verbose dfu serial --package feather_nrf52840_bootloader-0.4.0_s140_6.1.1.zip -p /dev/ttyACM0 -b 115200 --singlebank --touch 1200
```

Comment:
When using 0.4.1, the command is now as follows

```
adafruit-nrfutil --verbose dfu serial --package feather_nrf52840_express_bootloader-0.4.1_s140_6.1.1.zip -p /dev/ttyACM0 -b 115200 --singlebank --touch 1200
```

Reply:
Updated.